script.js

```diff
@@ -0,0 +1,149 @@
+const headerNavbar = document.getElementById('navbarSupportedContent');
+const resultBlock = document.getElementById('results');
+const pages = document.querySelector('.pagination');
+const searchButton = document.getElementById('search');
+const searchInput = document.querySelector('.form-control');
+const loader = document.querySelector('.loader');
```

Comment:
paste indentation between dom variables and variables

Reply:
Fixed